resources/services/cloudwatchlogs/log_groups.go

```diff
@@ -0,0 +1,115 @@
+package cloudwatchlogs
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func LogGroups() *schema.Table {
```

Comment:
Kindly add generation instruction here.
In this case, this should be something like this:
```go
//go:generate cq-gen --resource log_groups --config gen.hcl --output .
```

Reply:
Don't forget to install tools (`make install-tools`) before running generate on this file, too.

Reply:
I added the command in a commented out line like the above and `go generate` seems to work correctly... but it erases the comment line with the command. Is this expected or am I missing something? Thanks!

Reply:
This line should be left as is.
@hermanschaaf have you seen anything like this lately in cq-gen?

Reply:
@candiduslynx @deep-spaced Ah, you need to add a few lines to the top of the `gen.hcl` file for this comment to get generated:

```
service          = "aws"
output_directory = "."
add_generate     = true
```

Reply:
Perfect, that is super helpful! Added!